### PR TITLE
124 rest price updates

### PR DESCRIPTION
**Pull request by hornmo**

### CODE REVIEW

File: server/gokbg3/grails-app/services/gokbg3/RestMappingService.groovy [hunk 1]
```diff
@@ -151,8 +152,24 @@ class RestMappingService {
         }
         else {
           switch (p.type) {
+            case Float.class:
+              if (p.name == 'price') {
+                String pstring = obj[p.name] ? "${obj[p.name].trunc(2)}" : null
+
+                if (pstring) {
+                  if (!pstring.contains('.')) {
```
Comment (philboeselager) [inline server/gokbg3/grails-app/services/gokbg3/RestMappingService.groovy:160]: Are we sure that pstring does not contain other separator chars, esp. `,` ?

**Review by philboeselager — COMMENTED** [review]
1 comment, rest is fine.

File: server/gokbg3/grails-app/services/gokbg3/RestMappingService.groovy (hunk 1 above)
Comment (hornmo) [reply server/gokbg3/grails-app/services/gokbg3/RestMappingService.groovy:160]: I would think so, `Float.toString()` should always be of the form `123.23345`

Comment (philboeselager) [reply server/gokbg3/grails-app/services/gokbg3/RestMappingService.groovy:160]: Sure. I missed that.

**Review by philboeselager — APPROVED** [review]

**Pull request merged**